Project1/com.OrangeHRM.hr/src/test/java/TestNGAutomation/LaunchBrowser.java

```diff
@@ -15,7 +15,11 @@ public void launchChrome() {
 		driver = new ChromeDriver();
 		driver.manage().window().maximize();
 		driver.navigate().to("https://opensource-demo.orangehrmlive.com/");
-		
+		System.out.println("===========TrialText=============");

```

Comment:
This line of statement is not required here. Kindly review it and take respective action.